frontend/context/ServicesProvider.tsx

```diff
@@ -239,8 +238,8 @@ export const ServicesProvider = ({ children }: PropsWithChildren) => {
         services,
         serviceWallets,
         isError,
-        isFetched: isServicesFetched,
-        isLoading,
+        isFetched: !isServicesLoading,
```

Comment:
we need to change `isFetched` to `isLoading`, there are still places where the buttons stuck in infinite loading b/c service is not created and isFetched is always false due to the query having `enabled=false`